demos/mqtt/mqtt_demo_basic_tls/filePaths.cmake

```diff
@@ -0,0 +1,22 @@
+# This file is to add source files and include directories
+# into variables so that it can be reused from different repositories
+# in their Cmake based build system by including this file.
+#
+# Files specific to the repository such as test runner, platform tests
+# are not added to the variables.
+
+set( DEMO_NAME "mqtt_demo_basic_tls" )
```

Comment:
What is the advantage of separating these _demo-specific_ configurations from the `mqtt_demo_basic_tls/CMakeLists.txt` file? 
The comments mention about this file being re-used by different repositories. Which other repository is being referred to? The `amazon-freertos` repository will have a FreeRTOS specific demo instead of this one, as this will target POSIX platforms

Reply:
My suggestion would be to merge this within the `mqtt_demo_basic_tls/CMakeLists.txt` file to limit the Cmake files to one per demo